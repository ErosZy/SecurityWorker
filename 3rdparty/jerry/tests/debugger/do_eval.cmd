e a
n
eval a
break f
n
e b
next
e b
e "1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 XXX 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 YYY 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 ZZZ " + 123
e b = 8
n
e a
c
