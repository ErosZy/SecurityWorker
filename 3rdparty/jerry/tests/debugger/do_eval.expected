Connecting to: localhost:5001
Stopped at tests/debugger/do_eval.js:15
(jerry-debugger) e a
undefined
(jerry-debugger) n
Stopped at tests/debugger/do_eval.js:23
(jerry-debugger) eval a
5
(jerry-debugger) break f
Breakpoint 1 at tests/debugger/do_eval.js:19 (in f() at line:17, col:1)
(jerry-debugger) n
Stopped at breakpoint:1 tests/debugger/do_eval.js:19 (in f() at line:17, col:1)
(jerry-debugger) e b
undefined
(jerry-debugger) next
Stopped at tests/debugger/do_eval.js:20 (in f() at line:17, col:1)
(jerry-debugger) e b
6
(jerry-debugger) e "1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 XXX 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 YYY 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 ZZZ " + 123
1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 XXX 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 YYY 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 1234567890 ZZZ 123
(jerry-debugger) e b = 8
8
(jerry-debugger) n
Stopped at tests/debugger/do_eval.js:24
(jerry-debugger) e a
11.3
(jerry-debugger) c
